blaze/compute/tests/test_postgresql_compute.py

```diff
@@ -410,6 +424,17 @@ def test_distinct_on(sql):
     assert odo(computation, tuple) == (('a', 1), ('b', 2))
 
 
+def test_relabel_columns_over_selection(sql_more_data):
```

Comment:
for the postgresql test we should actually compute this and check the resulting data instead of just checking the sql string that it emits.